portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudIndividualTests.testcase

```diff
@@ -27,6 +27,53 @@ definition {
 		}
 	}
 
+	@description = "Set custom range filter with data for individual activity"
+	@priority = "5"
+	test CanSetDataWithCustomRangeOnIndividualActivitiesCard {
+		Navigator.openURL();
```

Comment:
Hi @calvinctvo , is `Navigator.openURL();` really needed?

I would suggest you check if it is really needed, and if yes, please move it and `ACUtils.addSiteAndPage();`
 to the setUp of this test case file, since both actions are applicable for other test cases placed on this file we can avoid repeating common actions into each test case body.

So you can also remove these calls from `CanSetEmptyDataCustomRangeOnIndividualActivitiesCard` test case.